science/gnss-sdr/Portfile

```diff
@@ -5,19 +5,19 @@ PortSystem          1.0
 PortGroup           cmake 1.0
 PortGroup           github 1.0
 PortGroup           active_variants 1.1
-PortGroup           cxx11 1.1
 PortGroup           compiler_blacklist_versions 1.0
 
 name                gnss-sdr
 maintainers         {michaelld @michaelld} {gmail.com:carles.fernandez @carlesfernandez} openmaintainer
-description         An Open Source Global Navigation Satellite Systems (GNSS)(for example: GPS, Galileo, Glonass, Beidou, etc) Software Defined Radio (SDR) Receiver
+description         An Open Source Global Navigation Satellite Systems (GNSS) (for example: GPS, Galileo, Glonass, Beidou, etc) Software Defined Radio (SDR) Receiver
 categories          science
 license             GPL-3
 platforms           darwin
 
 dist_subdir         gnss-sdr
 
 # Requires C++14
+compiler.cxx_standard 2014
```

Comment:
Please go ahead and remove the next line blacklisting ... it's not necessary with C++14 setting.